draft-irtf-cfrg-opaque.md

```diff
@@ -369,10 +369,10 @@ Output:
 - private_key, a private key.
 - public_key, the associated public key.
```

Comment:
#244 mentions the unification of capitalization in input argument comments, which is not in this PR. Is that planned for another PR?

Reply:
I think it is fine here, since we do not want to capitalize.